### PR TITLE
Feature/mypage

### PR DESCRIPTION
**Pull request by munyeol-Yoon**
리뷰안하면 죽음뿐

### CODE REVIEW

File: src/pages/MyPage/MyPage.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  const handleOnClickSideBarCategory = (component) => {
+    setDisplayComponent(component);
+  };
+
+  return (
+    <div>
+      <SideBar user={user} handleOnClickSideBarCategory={handleOnClickSideBarCategory} />
+      {displayComponent === 'recentVideos' && <RecentlyVideosComponent user={user} />}
+      {/* {displayComponent === 'chart' && <ChartComponent user={user} />} */}
```
Comment (leeejin) [inline src/pages/MyPage/MyPage.jsx]: 이거요!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Comment (eunkyum) [inline src/pages/MyPage/MyPage.jsx]: ✂️

File: src/zustand/recent-video.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        if (index !== -1) {
+          state.recent.splice(index, 1);
+        }
+
+        state.recent.unshift(data);
+
+        if (state.recent.length > 10) {
+          state.recent.pop();
+        }
+      });
```
Comment (leeejin) [inline src/zustand/recent-video.js:27]: 참 잘했어요 ! @

File: src/zustand/recent-video.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        const index = state.recent.indexOf(data);
+        if (index !== -1) {
+          state.recent.splice(index, 1);
+        }
+
+        state.recent.unshift(data);
+
+        if (state.recent.length > 10) {
+          state.recent.pop();
+        }
```
Comment (eunkyum) [inline src/zustand/recent-video.js:26]: 너무 잘했는걸요 ? 문열공주 짱짱👍🏻

**Review by leeejin — APPROVED** [review]
구두우우우우ㅜㅇ

**Review by eunkyum — APPROVED** [review]
문열공주짱짱 👍🏻👍🏻👍🏻

**Pull request merged**